crates/config/src/prover.rs

```diff
@@ -17,15 +18,20 @@ pub enum ProverType {
 }
 
 impl ProverRunnerConfig {
-    pub fn fill_values_with_prompt(self) -> anyhow::Result<ProverRunnerConfig> {
-        let prover = PromptSelect::new("Select Prover:", ProverType::iter()).ask();
+    pub fn fill_values_with_prompt(
+        self,
+        prev_atlantic_api: &str,
+    ) -> anyhow::Result<ProverRunnerConfig> {
+        let prover_type = PromptSelect::new("Select Prover:", ProverType::iter()).ask();
 
-        let url = match prover {
-            ProverType::Dummy => "".to_string(),
-            ProverType::Atlantic => Prompt::new("Input Atlantic prover API key:").ask(),
+        let url = match prover_type {
+            ProverType::Dummy => prev_atlantic_api.to_string(),
```

Comment:
This should be `"".to_string()`, right?

Reply:
If the user runs the CLI twice first with Dummy and then with Atlantic, the default value will indicate that he needs to input an `ATLANTIC_API_KEY`. The goal from this is to avoid writing an empty string when Dummy prover is selected because that field is not used in this case. However, if you think that it doesn't make much sense, I can simply change it for the empty string